RestWrite.js

```diff
@@ -654,16 +657,29 @@ RestWrite.prototype.runDatabaseOperation = function() {
     }
   }
 
+  function sendEmailVerification () {
```

Comment:
Is this the right place to define the `sendEmailVerification` function?
What if you want to call this manually on a parse job?
What if you want to disable email verification?

Maybe we abstract this to something like `validatePayload(className, payload)` and add validation rules for the _User model that are configurable?